vars/RedHat.yml

```diff
@@ -7,3 +7,39 @@ libvirt_vm_log_owner: qemu
 libvirt_vm_volume_creation_env:
   VOLUME_GROUP: qemu
   VOLUME_OWNER: qemu
+
+# Packages that are only necessary if you require EFI support
+libvirt_vm_extra_packages_efi:
+  - edk2.git-ovmf-x64 # Official OVMF package doesn't boot (CentOS 7.5)
```

Comment:
Interesting OVMF was just proposed for addition to the nova containers in kolla: https://review.openstack.org/608579